spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/BindingProperties.java

```diff
@@ -54,6 +54,16 @@
 	public static final String OUTPUT_TYPE_KEY = OUTPUT_BINDING_KEY_PREFIX + CONTENT_TYPE_KEY;
 
 	/**
+	 * Group property key for input.
+	 */
+	public static final String INPUT_GROUP_KEY = INPUT_BINDING_KEY_PREFIX + "group";
+
+	/**
+	 * Subscription durability for input.
+	 */
+	public static final String INPUT_DURABLE_SUBSCRIPTION_KEY = INPUT_BINDING_KEY_PREFIX + "durableSubscription";
```

Comment:
I think this property needs to be added in `spring-cloud-stream`'s [BindingProperties] (https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingProperties.java) so that the it gets updated as `ConsumerProperties` when binding.